views/home.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/styles.css">
+    <link rel="icon" href="/public/favicon.ico">
+    <title>Tomás Belmar</title>
+</head>
+
+<body>
+ <div class="nav">
+    <a ref="about.html" href="./about.html">About</a>
+    <a ref="contact.html" href="./contacts.html">Contacts</a>
+
+ </div>   
+<div class="top-container">
+    <img class="top-cloud" src="/public/cloud.png" alt="cloud">
```

Comment:
And also, on every image that you have here, you need to remove the `/public` part of the path also. Otherwise it will not appear on your browser.